{"name": "Проверяем, что нельзя создать пользователя, который уже зарегистрирован, запрос возвращает ошибку с кодом 403 Forbidden", "status": "passed", "start": 1731761852162, "stop": 1731761852572, "uuid": "6d12cd85-13c9-4f6a-adfb-86b4936cab49", "historyId": "46fe8ae2de49412e05d3eb718ab28fcd", "testCaseId": "46fe8ae2de49412e05d3eb718ab28fcd", "fullName": "tests.test_create_user.TestCreateUser#test_create_existing_user_code_403", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_user"}, {"name": "subSuite", "value": "TestCreateUser"}, {"name": "host", "value": "hanastasia-mbby"}, {"name": "thread", "value": "1011-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_user"}]}